modules/app/components/layout/Footer/NewFooter.tsx

```diff
@@ -0,0 +1,235 @@
+        {
+          url: 'https://migrate.makerdao.com/',
+          title: t('Migrate Dashboard')
+        },
+        {
+          url: 'https://makerburn.com/',
+          title: t('MakerBurn')
+        },
+        {
+          url: 'https://daistats.com/',
+          title: t('DAI Stats')
+        }
+      ]
+    },
+    {
+      header: t('Developer'),
+      list: [
+        {
+          url: 'https://makerdao.com/whitepaper',
+          title: t('Whitepaper')
+        },
+        {
+          url: 'https://docs.makerdao.com/',
+          title: t('Technical Docs')
+        },
+        {
+          url: 'https://vote.makerdao.com/api-docs',
+          title: 'API Docs'
+        },
+        {
+          url: 'https://github.com/makerdao/developerguides',
+          title: 'Developer Guides'
+        },
+        {
+          url: 'https://www.notion.so/makerdao/Maker-Brand-ac517c82ff9a43089d0db5bb2ee045a4',
+          title: t('Brand Assets')
+        },
+        {
+          url: 'https://makerdao.com/en/feeds/',
+          title: t('Oracle Feeds')
+        }
+      ]
+    }
+  ];
+
+  const logos = {
+    makerdao: [
+      { title: 'Discord', url: 'https://chat.makerdao.com', icon: 'discord' },
+      { title: 'Twitter', url: 'https://twitter.com/MakerDAO', icon: 'twitter' },
+      { title: 'Reddit', url: 'https://www.reddit.com/r/MakerDAO/', icon: 'reddit' },
+      { title: 'Telegram', url: 'https://t.me/makerdaoOfficial', icon: 'telegram' },
+      { title: 'YouTube', url: 'https://www.youtube.com/MakerDAO', icon: 'youtube' },
+      { title: 'GitHub', url: 'https://www.github.com/makerdao', icon: 'github' }
+    ],
+    makerdux: [
+      { title: 'Discord', url: 'https://discord.gg/GHcFMdKden', icon: 'discord' },
+      { title: 'Twitter', url: 'https://twitter.com/MakerDUX', icon: 'twitter' },
+      { title: 'GitHub', url: 'https://github.com/makerdao/governance-portal-v2', icon: 'github' },
+      { title: 'Canny', url: 'https://makergovernance.canny.io/', icon: 'canny' },
+      { title: 'Immunifi', url: 'https://immunefi.com/bounty/makerdao/', icon: 'immunifi' }
+    ]
+  };
+  return (
+    <div sx={{ position: 'relative', width: '100vw' }}>
+      <div
+        sx={{
+          pt: '100%',
+          width: '100%',
+          zIndex: -1,
+          position: 'absolute',
+          transform: 'rotate(21deg)',
+          backgroundImage,
+          backgroundSize: '100%',
+          backgroundRepeat: 'no-repeat'
+          //WIP mobile settings
+          // pt: '200%',
+          // width: '200%',
+          // // left: -300,
+          // right: -200,
+          // // top: 300,
+          // // pt: '500px',
+          // zIndex: -1,
+          // position: 'absolute',
+          // transform: 'scale(1.5)',
+          // // transform: 'rotate(21deg)',
+          // backgroundImage,
+          // // backgroundSize: '300%',
+          // backgroundRepeat: 'no-repeat',
+          // backgroundPosition: 'center bottom'
```

Comment:
Still WIP? ✂️

Reply:
ah weird, NewFooter should have been deleted, I renamed the file to LandingFooter. Nice catch